### ISSUE TITLE
Select the previous (not current) editor by default in "Show all editors by most recently used"

### ISSUE THREAD

**Issue by joeshaw** [microsoft/vscode#113918]
VSCode 1.52.1 on macOS 11.1

If you open the "Show all editors by most recently used" picker (`workbench.action.showAllEditorsByMostRecentlyUsed`), the currently active editor is always the default choice.  This is almost certainly never what you really want, since hitting enter would be a no-op.

I would like for the previous most recently active editor to be the default choice, so I could jump back and forth between these two files easily.  (I have this bound to `C-x b` to emulate Emacs; implementing this would also emulate its behavior.)

I'd also be satisfied with this being an option if it's not the default.

**Comment by joeshaw** [comment]
The quick-and-easy way to do this would appear to be to change https://github.com/microsoft/vscode/blob/c03bc427cf71dae7766cd6e451c492b12e87e544/src/vs/workbench/browser/parts/editor/editor.contribution.ts#L306-L312 to add something like:

```
if (entries.length > 1) {
    const mru = entries.shift();
    entries.push(mru);
}
```

before the `return entries;` line.

**Comment by bpasero** [comment]
Related: https://code.visualstudio.com/docs/editor/editingevolved#_common-questions

This is for the file/editor picker (default `Cmd+P`) and not the picker you talk about but maybe that is sufficient.

**Comment by joeshaw** [comment]
You're right, `workbench.action.quickOpenPreviousRecentlyUsedEditor` does appear to do what I want.  I was under the misimpression that it would automatically switch the editor, as that's what appears to happen when it's bound to `ctrl+tab`.  Thank you for pointing this out!

**Comment by joeshaw** [comment]
Oop, no, sorry, this doesn't quite do what I want.  It doesn't allow you to type in a name of a different open editor.

**Comment by ArturoDent** [comment]
You could make a macro to do this pretty easily.  Just using these two commands:

        "workbench.action.showAllEditorsByMostRecentlyUsed",
        "workbench.action.quickOpenNavigateNext"

Then that quick open panel will open with the second item selected.

**Comment by joeshaw** [comment]
@ArturoDent What macro extension do you recommend?

**Comment by ArturoDent** [comment]
I use [multi-command](https://marketplace.visualstudio.com/items?itemName=ryuta46.multi-command) although there are more powerful macro extensions this one is easy to work with.
